src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -271,6 +271,8 @@ Result AudioStreamAAudio::open() {
 
     if (mLibLoader->builder_setIsContentSpatialized != nullptr) {
         mLibLoader->builder_setIsContentSpatialized(aaudioBuilder, mIsContentSpatialized);
+    } else {
```

Comment:
isContentSpatialized refers to the content itself, not a feature of Android. You can play spatialized audio on old versions of Android if the content is already spatialized.  So maybe it it is OK to just set mIsContentSpatialized to whatever the app says. On old versions it will be ignored. On new versions of Android AAudio will care about the flag.